examples/react-app/src/Avatars.tsx

```diff
@@ -1,28 +1,55 @@
 import { PlmgAvatar } from '@ducky/plumage-react';
 
-const styles = {
-  marginLeft: '-15px',
-};
+const users = [
+  {
+    name: 'John Foo',
+    picture: 'https://static.ducky.eco/icons/maskable_icon_192.png',
+    user_id: '1',
+  },
+  {
+    name: 'Mary Berry',
+    picture: '',
+    user_id: '2',
+  },
+  {
+    name: '',
+    picture: '',
+    user_id: '3',
+  },
+  {
+    name: 'Carl Carlson',
+    picture: 'https://static.ducky.eco/icons/maskable_icon_192.png',
+    user_id: '4',
+  },
+  {
+    name: 'Ima Another',
+    picture: '',
+    user_id: '5',
+  },
+  {
+    name: '',
+    picture: '',
+    user_id: '6',
+  },
+];
+
+const avatarsComponents = [
+  users.map(({ name, picture, user_id }) => (
+    <PlmgAvatar
+      key={user_id}
+      userDeleted={name === '' ? true : false}
+      imageUrl={picture}
+      size={'medium'}
+      style={user_id !== '1' ? { marginLeft: '-15px' } : null}
+    />
+  )),
+];
```

Comment:
I know this isn't production code per se, so it is not critical:
1. This results in a nested array, so `[[<Component/>, <Component/>]]`, I'd get rid of the surrounding array
2. `name === '' ? true : false` is the same as `name === ''`
3. Rather than marginLeft, I think we would get the same result with `gap: -15px` on the flex div and avoid the ternary

Reply:
1, 2 - great. thanks for corrections
3 - gap won't take a negative value. I've taken out the ternary and added a margin-left on the container div to negate the positioning of the negative margin on the first flex item.

Reply:
You're absolutely right, my bad